src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
 import './FindMovie.scss';
 
+import { request } from '../../api/api';
+
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
+export const FindMovie = ({ addMovie }) => {
+  const [searchField, setSearchField] = useState('');
+  const [searchFieldError, setsearchFieldError] = useState(false);
+  const [movieFromServer, setMovieFromServer] = useState(null);
+
+  const findMovie = (movie) => {
+    request(movie)
+      .then((result) => {
+        if (result.Response !== 'False') {
+          return setMovieFromServer(result);
+        }
+
+        return Promise.reject(result.Error);
+      })
+      .catch(() => {
+        setsearchFieldError(true);
+      });
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames('input', { 'is-danger': searchFieldError })}
+              value={searchField}
+              onChange={(event) => {
+                setSearchField(event.target.value);
+                setsearchFieldError(false);
+              }}
+            />
+          </div>
+
+          {searchFieldError
+        && (
         <p className="help is-danger">
           Can&apos;t find a movie with such a title
         </p>
-      </div>
-
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+        )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={() => findMovie(searchField)}
```

Comment:
use onSumbmit for that,  in that way user can press the 'enter' key and trigger submit an event